packages/cli/src/menus/shared.ts

```diff
@@ -1,10 +1,16 @@
 /* eslint-disable no-console */
+import { Address, CredentialType } from "@blaze-cardano/core";
 import { input, select } from "@inquirer/prompts";
 import { AssetAmount, type IAssetAmountMetadata } from "@sundaeswap/asset";
-import { ADA_METADATA } from "@sundaeswap/core";
+import {
+  ADA_METADATA,
+  EContractVersion,
+  type IPoolByAssetQuery,
+  type IPoolData,
+} from "@sundaeswap/core";
 import packageJson from "../../package.json" assert { type: "json" };
 import type { State } from "../types";
-import { prettyAssetId } from "../utils";
+import { getPoolData, prettyAssetId } from "../utils";
```

Comment:
Reminder: File extensions need to include explicit extensions since this is a module repo. i.e. `../utils.js`

It's probably not an issue since this runs via command line and is not bundled in a project, but for the sake of consistency, please ensure all files in the CLI package follow this convention.